README.md

```diff
@@ -652,6 +652,10 @@ Good:
 </View>
 ```
 
+### Children Components Not Re-Rendering
+Have them outside the MapView component and position absolute.
```

Comment:
I would make the explanation a little more verbose so people don't have to go to the linked issue (but keep the link though)

Thanks

Reply:
I have explained things more clearly, hope this is suffice!

Reply:
that's great 😻